test/stringinterface-test.jl

```diff
@@ -0,0 +1,14 @@
+function test_stringinterface()
+    print("stringinterface...")
```

Comment:
The test file has `stringinterface` in its name, but it actually tests the function `call_interpreter` -- wouldn't it make more sense to align these names, for better grepability?

Reply:
It is the "interpreter string interface". What name do you suggest? `test_call_interpreter`?